package.json

```diff
@@ -36,10 +36,10 @@
     "build:src": "webpack --config webpack-main.config.js",
     "build:test": "webpack --config webpack-test.config.js",
     "build": "npm run build:libs && npm run build:src && npm run build:test",
-    "lint:js": "eslint src/main/webapp/dev -c static-analysis/teammates-eslint.yml --ext es6",
-    "lint:json": "eslint src/main/webapp/data/*.json src/main/resources/*.json src/test/resources/data/*.json -c static-analysis/teammates-eslint-json.yml",
-    "lint:css": "stylelint src/main/webapp/stylesheets/*.css --config static-analysis/teammates-stylelint.yml",
-    "lint:spaces": "lintspaces -n -t -d spaces -l 1 -. \"src/**/*.jsp\" \"src/**/*.tag\" \"src/main/**/*.html\" \"src/**/*.xml\" \"src/**/*.json\" \"src/**/*.css\" \"src/**/*.java\" \"src/**/*.es6\" \"src/**/*.properties\" \"*.yml\" \"*.gradle\" \"static-analysis/*.*ml\"",
+    "lint:js": "eslint *.js src/main/webapp/dev/**/*.es6 -c static-analysis/teammates-eslint.yml",
```

Comment:
Does this mean it'll lint all the generated `.js` files too (if we run `npm run lint` after `npm run build`?